test/unit/server/request_airdrop_test.js

```diff
@@ -33,7 +35,7 @@ describe('Server#requestAirdrop', function() {
 
   it('returns true when the account is created', function(done) {
     const friendbotUrl = 'https://friendbot.stellar.org';
-    const accountId = "GBZC6Y2Y7Q3ZQ2Y4QZJ2XZ3Z5YXZ6Z7Z2Y4QZJ2XZ3Z5YXZ6Z7Z2Y4";
+    const accountId = 'GBZXN7PIRZGNMHGA7MUUUF4GWPY5AYPV6LY4UV2GL6VJGIQRXFDNMADI';
```

Comment:
Can you review/merge #31 before/after this so we get consistency? Otherwise, this will be dependent on who checks out the repo, right?